src/yb/master/catalog_manager.h

```diff
@@ -463,6 +463,9 @@ class CatalogManager : public tserver::TabletPeerLookupIf,
   Status GetColocatedTabletSchema(const GetColocatedTabletSchemaRequestPB* req,
                                   GetColocatedTabletSchemaResponsePB* resp);
 
+  Status ListTableInfo(const ListTablesRequestPB* req,
```

Comment:
This has the same signature as the one below? why are two separate ones needed.

Please consider merging them.